backend/app/Http/Controllers/Web/BackupController.php

```
@@ -0,0 +1,597 @@
+        return $dir;
+    }
+
+    /**
+     * Create a full backup (database + uploads).
+     */
+    public function createFullBackup(Request $request)
+    {
+        try {
+            $filename = $this->runBackupInternal(false);
+
+            AuditLog::create([
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\AuditLog::create().


(staticMethod.notFound)